native/Cargo.toml

```diff
@@ -3,17 +3,17 @@ name = "ctrlc-windows"
 version = "0.1.0"
 authors = ["Frontside Engineering <engineering@frontside.com>"]
 license = "MIT"
-build = "build.rs"
 edition = "2018"
 exclude = ["artifacts.json", "index.node"]
 
 [lib]
 name = "ctrlc_windows"
 crate-type = ["cdylib"]
 
-[build-dependencies]
-neon-build = "0.5.0"
```

Comment:
With the updated version, you either use this _or_ the cli. We are using the cli vs `build.js`, so removing this.